src/include/sof/lib_manager.h

```diff
@@ -84,6 +84,8 @@ struct ext_library {
 	uint32_t mods_exec_load_cnt;
 	struct ipc_lib_msg *lib_notif_pool;
 	uint32_t lib_notif_count;
+
+	void *runtime_data;
```

Comment:
the commit message leaves me scratching my head

a)  "We cannot assume that the GEN bit is already enabled": who is 'we'? is this firmware or kernel?

b)  "The first LOAD_LIBRARY with lib_id = 0 and valid dma_id 0". what is the definition of 'first'? Is this for the basefw?

c) "Second LOAD_LIBRARY with valid lib_id and dma_id". What is the definition of 'second'? Is this for the first library that's not the basefw? Is this that the kernel is *REQUIRED* to send two LOAD_LIBRARY IPCs with the lib_id being zero first and something non-zero next?

c) Single stage flow (currently supported)
What is the point of having two options, what is the recommended solution and why? This mixes current situation and suggested improvement, it's very hard to follow your line of thought

Reply:
I will drop this patch